src/llmcompressor/transformers/finetune/text_generation.py

```diff
@@ -224,6 +228,13 @@ def initialize_model_from_path(
         "trust_remote_code": model_args.trust_remote_code_model,
     }
     # this calls from_pretrained under the hood so should be FSDP safe
+
+    # optimized models must be decompressed to carry out oneshot/train/etc
+    if is_model_quantized_from_path(model_path):
```

Comment:
Is this helper `is_model_quantized_from_path` really necessary? What happens if you attempt to load a non-quantized model with `quantization_config=CompressedTensorsConfig(run_compressed=False)`?

Reply:
Good question, we do have is_model_quantzied in CT, but inputs a module. Here you check the config without loading. 
If you load a non-quantized model with the config, then It would error out -- but here on load it only adds the quant_config if its quantized. Its not a case we should worry about.